app/scripts/lib/experiment.js

```diff
@@ -114,7 +114,7 @@ define(function (require, exports, module) {
         return;
       }
 
-      for (let experimentName in this._allExperiments) {
+      for (const experimentName in this._allExperiments) {
```

Comment:
Is this what you mean: https://codepen.io/vladikoff/pen/MJqLjK?editors=1111 

Seems fine, see console output